lib/configure.js

```diff
@@ -175,124 +174,74 @@ function configure(appName, opts, dtProbes) {
- * and to set the logging level appropriately.
+ * requested.
  * @returns {Object} A bunyan logger object
  */
 function configureLogging(appName, bunyanCfg, verbose) {
+    assert.optionalObject(bunyanCfg, 'config.bunyan');
+
     var level;
     if (bunyanCfg) {
-        level = process.env.LOG_LEVEL || bunyanCfg.level || 'info';
+        assert.optionalString(bunyanCfg.level, 'config.bunyan.level');
+
+        level = bunyan.resolveLevel(process.env.LOG_LEVEL ||
+                    bunyanCfg.level || 'info');
     } else {
-        level = process.env.LOG_LEVEL || 'info';
+        level = bunyan.resolveLevel(process.env.LOG_LEVEL || 'info');
     }
-    var log = bunyan.createLogger({
-        name: appName,
-        streams: [ {
-            level: level,
-            stream: process.stderr
-        } ],
-        serializers: restify.bunyan.serializers
-    });
 
-    level = log.level();
-
-    /*
-     * This is ugly, but we set this up so that if muskie is invoked with a
-     * -v flag, then we know you're running locally, and you just want the
-     * messages to spew to stderr. Otherwise, it's "production", and muskie
-     * likely logs to a syslog endpoint.
-     */
-    if (verbose || !bunyanCfg) {
-        if (verbose) {
-            level = Math.max(bunyan.TRACE, (log.level() - verbose.length * 10));
-            log.level(level);
-        }
-
-        /*
-         * If the configured logging level is at or below DEBUG then enable
-         * source logging.
-         */
-        if (level <= bunyan.DEBUG) {
-            log = log.child({src: true});
-        }
-
-        return (log);
+    if (verbose) {
+        level = Math.max(bunyan.TRACE, (level - verbose.length * 10));
     }
 
-    assert.object(bunyanCfg, 'config.bunyan');
-    assert.optionalString(bunyanCfg.level, 'config.bunyan.level');
-    assert.optionalObject(bunyanCfg.syslog, 'config.bunyan.syslog');
-
-    level = bunyan.resolveLevel(process.env.LOG_LEVEL ||
-                                bunyanCfg.level ||
-                                'info');
-
-    log.debug('Bunyan log level: ' + level);
+    var streams = [];
 
-    if (bunyanCfg.syslog) {
-        var streams = [];
-        var sysl;
-
-        assert.string(bunyanCfg.syslog.facility,
-                      'config.bunyan.syslog.facility');
-        assert.string(bunyanCfg.syslog.type, 'config.bunyan.syslog.type');
-
-        sysl = bsyslog.createBunyanStream({
-            facility: bsyslog.facility[bunyanCfg.syslog.facility],
-            name: appName,
-            host: bunyanCfg.syslog.host,
-            port: bunyanCfg.syslog.port,
-            type: bunyanCfg.syslog.type
-        });
-
-        streams.push({
+    streams.push({
             level: level,
-            stream: sysl
-        });
+            stream: process.stderr
+    });
 
+    if (level >= bunyan.INFO) {
         /*
          * We want debug info only IFF the request fails AND the configured
```

Comment:
Might as well fix this typo while you're in here?

Reply:
TIL. Sorry about that.

This can be reverted then.

Reply:
The wording is a bit odd to me. I think it's the *only*. I'll reword it a bit.